teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
-        <h3>Instructions</h3>
-        <p>Read through the scope and sequence of the curriculum, skim the lesson plans, and answer the following questions.</p>
+    <div class="block block__flush-top">
+        <h1>Curriculum Review</h1>
+        <p>Middle School Example Curriculum</p>
+        <p>Start the review by selecting a dimension. You do not need to complete all dimensions in one sitting. You’ll be able to download a dimension report for each dimension as well as a summary report at the end.</p>
+        <p>
+            <a href="#">
+                Content<br>
+                Covers core knowledge and skills in content standards<br>
+                6 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Utility<br>
+                Supports effective teaching<br>
+                7 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Quality<br>
+                Accurate and well presented<br>
+                5 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Efficacy<br>
+                Improves financial knowledge, skills, or behaviors<br>
+                3 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
     </div>
-    <div class="block">
-        <h3>Criterion 1: Name of this criterion</h3>
-        <p>Does the curriculum address grade-level appropriate topics for saving and investing? Select yes if the following components are addressed. Select no if the following components are not addressed.</p>
-        <ol>
-            <li>
-                <h4>Indicator</h4>
+    <div class="block
+                block__flush-top
+                block__flush-bottom">
+        <h2 class="h1">{# {% include icons/document-round.svg %} #}Content</h2>
+        <div class="lead-paragraph
+                    u-mb30">
+            <p>This dimension assesses whether the curriculum content helps students develop knowledge, skills, and behaviors that are important for financial capability. Evaluation criteria are based on research and major national and state education standards.</p>
+        </div>
+        <h3 class="h2">Instructions</h3>
+        <ul>
+            <li>Read through the scope and sequence of the curriculum.</li>
+            <li>Skim the lesson plans.</li>
+            <li>Select <strong><em>yes</em></strong> for the components that are addressed, and <strong><em>no</em></strong> for those that are not.</li>
```

Comment:
I'm not certain, but I've heard that some screenreaders will read `strong` and `em` with extra emphasis. The `em` uses seem legit, but let's use `<b>` instead of `<strong>` here to ensure things aren't _extra_ emphasized by screenreaders.

Reply:
Fixed